index.html

```diff
@@ -0,0 +1,185 @@
+      <h1>I'm Ijeoma Odinko, Glad to see you!</h1>
+      <p class="header-text">I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+    </div> 
+    <div class="about">
+        <a href="#">LET'S CONNECT</a>
+    </div>
+    <div class="image">
+      <ul>
+          <li><a href="#"><i class="icons fab fa-twitter" aria-hidden="true"></i></a></li>
+          <li><a href=""><i class="icons fab fa-linkedin-in" aria-hidden="true"></i></a></li>
+          <li><a href="#"><i class="icons fab fa-medium-m" aria-hidden="true"></i></a></li>
+          <li><a href=""><i class="icons fab fa-github" aria-hidden="true"></i></a></li>            <li><a href="#"><i class="icons fab fa-angellist" aria-hidden="true"></i></a></li>
+      </ul>
+    </div>
+    </section>
+    <!--section 1 of the portfolio card starts-->
+    <section class="sectiontemplate">
+      <img class="img1" src="img/Portfolio1.svg" alt="gym">
+      <div class="Section2Text">
+      <h2 class="face">Facebook 360</h2>
+        <ul class="list1">
+          <li class="can">Canopy</li>
+          <li class="special">Back End Dev</li>
+          <li class="special">2015</li>
+        </ul>
+      <p>A daily selection of privately personalized reads; 
+      no accounts or sign-ups required.</p>
+        <ul class="list2">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul> 
+    <button class="btn">See project</button>
+    </div>
+    </section>
+    <!--section 2 of the portfolio cards starts-->
+    <section class="sectiontemplate">
+      <img class="img1" src="img/Portfolio2.svg" alt="gym">
+      <div class="Section3Text">
+        <h2>Uber Navigation</h2> 
+        <ul class="list1">
+          <li class="can">Canopy</li>
+          <li class="special">Back End Dev</li>
+          <li class="special">2015</li>
+        </ul>
+      <p>A daily selection of privately personalized reads; 
+        no accounts or sign-ups required.</p>
+        <ul class="list3">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+      <button class="btn">See project</button>
+      </div>
+    </section>
+    <!--section 3 of the portfolio card starts-->
+    <section class="sectiontemplate">
+      <img class="img1" src="img/Portfolio3.svg" alt="gym">
+      <div class="SectionText">
+        <h2>Tonic</h2>
+        <ul class="list1">
+          <li class="can">Canopy</li>
+          <li class="special">Back End Dev</li>
+          <li class="special">2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no 
+          accounts or sign-ups required.</p>
+        <ul class="list2">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul> 
```

Comment:
- [ ] Take a look at this image below, kindly match your design with the Figma one
<img width="389" alt="width" src="https://user-images.githubusercontent.com/79058364/159116562-cb6831fa-3788-4f1b-adb9-944ee7c08d04.png">

- It should be like this instead ⏬ 
<img width="408" alt="oriwd" src="https://user-images.githubusercontent.com/79058364/159116642-3994e947-bc10-4428-ba3d-80019f376975.png">